.github/workflows/scripts/aot-demo.sh

```diff
@@ -17,7 +17,7 @@ function build-and-smoke-test-android-aot-demo {
 
     rm -rf taichi-aot-demo
     # IF YOU PIN THIS TO A COMMIT/BRANCH, YOU'RE RESPONSIBLE TO REVERT IT BACK TO MASTER ONCE MERGED.
-    git clone https://github.com/taichi-dev/taichi-aot-demo
+    git clone https://github.com/bobcao3/taichi-aot-demo
```

Comment:
...?

Reply:
Have to do this for now because this changes an API that the aot demo uses, and the workflow is to do this first, then merge the change in the aot demo repo, and then make another PR here to change it back